test/jdk/security/infra/java/security/cert/CertPathValidator/certification/AmazonCA.java

```diff
@@ -100,76 +100,66 @@ class AmazonCA_1 {
             "843t4By6YT/PVlePU2PCWejkrJQnKQAPOov7IA8kuO2RDWuzE/zF6Hotdg==\n" +
             "-----END CERTIFICATE-----";
 
-    // Owner: CN=good.sca1a.amazontrust.com, O=Amazon Trust Services, L=Seattle, ST=Washington, C=US, \
-    // SERIALNUMBER=5846743, OID.2.5.4.15=Private Organization, OID.1.3.6.1.4.1.311.60.2.1.2=Delaware, \
-    // OID.1.3.6.1.4.1.311.60.2.1.3=US
+    // Owner: CN=good.sca1a.amazontrust.com
     // Issuer: CN=Amazon, OU=Server CA 1A, O=Amazon, C=US
-    // Serial number: 703e4e4bbd78e2b6db5634f36c4ee944cb1a4
-    // Valid from: Mon Jul 29 16:53:36 PDT 2019 until: Sat Aug 29 16:53:36 PDT 2020
+    // Serial number: 75a5dd4b767bedc94a4239da65ed9dfef8218
+    // Valid from: Fri Dec 17 12:21:50 PST 2021 until: Tue Jan 17 12:21:50
+    // PST 2023
```

Comment:
Can you put the "PST 2023" at the end of the previous line instead of on a new line? I think it would be more readable. Same comment applies to all the test certificates.